apps/client/src/pages/create.tsx

```diff
@@ -511,6 +512,47 @@ class Create extends React.Component<CreateProps, CreateState> {
                     to try it out.
                   </FormText>
                 </FormGroup>
+                <FormGroup>
```

Comment:
Does one need a dedicated form group for V2? Or could one validate the uploaded model and set a `ModelType` according to that?